src/application/components/BannerAlert.tsx

```diff
@@ -37,7 +37,7 @@ export function BannerAlert() {
             }
           )}
         >
-          {banner.type === "loading" && <LoadingSpinner size="2xsmall" />}
+          {banner.type === "loading" && <Spinner size="xs" />}
```

Comment:
I took this opportunity to swap out the loading spinner for the orbit themed spinner